yb-voyager/cmd/importData.go

```diff
@@ -160,11 +160,17 @@ func startExportDataFromTargetIfRequired() {
 	importTableList := getImportTableList(tableListExportedFromSource)
 
 	lockFile.Unlock() // unlock export dir from import data cmd before switching current process to ff/fb sync cmd
+
+	if tconf.SSLMode == "prefer" || tconf.SSLMode == "allow" {
+		utils.PrintAndLog(color.RedString("Warning: SSL mode '%s' is not supported for 'export data from target' yet. Downgrading it to 'disable'.\nIf you dont want these settings you can restart the 'export data from target' with the --target-ssl-mode flag.", source.SSLMode))
+		tconf.SSLMode = "disable"
+	}
 	cmd := []string{"yb-voyager", "export", "data", "from", "target",
 		"--export-dir", exportDir,
 		"--table-list", strings.Join(importTableList, ","),
 		fmt.Sprintf("--transaction-ordering=%t", transactionOrdering),
 		fmt.Sprintf("--send-diagnostics=%t", callhome.SendDiagnostics),
+		"--target-ssl-mode", tconf.SSLMode,
```

Comment:
Should we also add root-cert arg to export-data-from-target? 
What happens if 
- someone uses "prefer" in import-data-to-target, with no value for target-root-cert flag
- Now, after cutover, export-data-from-target starts with sslmode=disable. 
- User is not happy with this behaviour, and wants to change sslmode to "require". But now, that will not work because there is no way to specify root-cert and root-cert is mandatory for yb cdc connector